korangar/src/world/model/node.rs

```diff
@@ -49,41 +50,48 @@ impl Node {
         current_rotation.into()
     }
 
-    pub fn world_matrix(&self, transform: &Transform, client_tick: ClientTick) -> Matrix4<f32> {
-        let animation_rotation_matrix = match self.rotation_keyframes.is_empty() {
-            true => Matrix4::identity(),
-            false => self.animaton_matrix(client_tick),
-        };
-
-        let rotation_matrix = Matrix4::from_angle_z(-transform.rotation.z)
-            * Matrix4::from_angle_x(-transform.rotation.x)
-            * Matrix4::from_angle_y(transform.rotation.y);
+    pub fn world_matrix(&self, client_tick: ClientTick, parent_matrix: &Matrix4<f32>, is_static: bool) -> Matrix4<f32> {
+        if is_static == true {
```

Comment:
`if foo == true` is very much not idiomatic. It should just be `if foo`.

I am surprised that clippy is not complaining about this.

Reply:
Now using match for this kind of comparison.